tests/deal_ticket/test_basic_submit.py

```diff
@@ -0,0 +1,85 @@
+import pytest
+from playwright.sync_api import Page, expect
+from vega_sim.service import VegaService
+from datetime import datetime, timedelta
+
+order_size = "order-size"
+order_price = "order-price"
+place_order = "place-order"
+order_side_sell = "order-side-SIDE_SELL"
+market_order = "order-type-Market"
+tif = "order-tif"
+expire = "expire"
+
```

Comment:
what abut coverage for FOK, GFN ?

Reply:
The tests in cypress only had GTT. But I can add the others

Reply:
just updated